Round Robin - JNT (#153)
jaybee84/ml-in-rd: content/04.model-complexity.md

Resampling without replacement can generate confidence intervals for the model predictions by iteratively exposing the models to incomplete datasets, mimicking real world cases where most rare disease datasets are incomplete [@doi:10.3390/genes11020226].
Resampling with replacement, or bootstrapping, helps estimate population values from datasets of limited size, and is also commonly used to find robust models when multiple models are combined into an ensemble ([@doi:10.1080/01621459.1997.10474007; @doi:10.1023/A:1010933404324; @doi:10.1198/0003130043277; @doi:10.1016/s0925-2312(01)00650-6; @doi:10.1016/j.neucom.2004.11.017; @doi:10.1002/sim.4780111607]).
Ensemble learning methods like random forests use _bagging_ (bootstrap aggregation) of independent decision trees that use similar parameters but different paths to form a consensus about the important predictive features [@doi:10.1023/A:1010933404324; @doi:10.1186/1472-6947-13-134; @doi:10.1214/aos/1031689014; @doi:10.1177/2045894019890549; @doi:10.1016/s0031-3203(02)00169-3]. 

> Regular ensemble learning has shown limited success in rare disease datasets where the label-uncertainty can be high. [TODO: citation?] 

Reviewer: ```suggestion
Regular ensemble learning has shown limited success in rare disease datasets with substantial label-noise. [TODO: citation?] 
```

Reviewer: @jaybee84 did you have a citation in mind for this?
